django-web-app/templates/login.html

```diff
@@ -34,20 +33,16 @@ <h1 class="head">Login</h1>
                             <label class="sub-head" for="username">
                                 Username
                             </label>
-                            <!-- <img class="input-img" src="/static/img/user2.png" alt="user"> -->
-<!--                             <i class='bx bxs-user input-img'></i> -->
-                            <input class="form-f" id="username" name="username" required  placeholder="Type your username" type="text">
-                            
+                            <input class="form-f" id="username" name="username" required placeholder="Type your username" type="text">
                         </div>
     
                         <div class="form-c-1">
                             <label class="sub-head" for="password">
                                 Password
                             </label>
-                            <!-- <img class="input-img" src="/static/img/padlock.png" alt="lock"> -->
-<!--                             <i class='bx bxs-lock input-img'></i> -->
                             <input class="form-f" id="password" name="password" placeholder="Type your Password" type="password" required>
-    
```

Comment:
Add an eye icon, along with cross eye, 
just like you see in most web apps, 
rest seems fine. 

add CSS accordingly.